package/upgrade/upgrade_manifests.sh

```diff
@@ -1239,6 +1239,28 @@ reuse_vlan_cn() {
   kubectl get clusternetwork vlan -o yaml | yq '.metadata.finalizers = []' | kubectl apply -f -
 }
 
+sync_containerd_registry_to_rancher() {
+  echo "Sync containerd-registry setting to Rancher"
+
```

Comment:
Currently we do not have version controls in the upgrade script, so would prefer to check if the `clusters.provisioning` registry is not empty and returns directly.